web/app/js/components/util/ApiHelpers.jsx

```diff
@@ -97,6 +97,7 @@ const ApiHelpers = (pathPrefix, defaultMetricsWindow = '1m') => {
   };
 
   const urlsForResource = (type, namespace) => {
```

Comment:
we could rename `urlsForResource` to `statUrls` if that would be more helpful than adding the comment